cudamapper/src/overlapper_triggered.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include <algorithm>
+#include <cstdio>
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+#include "overlapper_triggered.hpp"
+#include "cudamapper/overlapper.hpp"
+#include "matcher.hpp"
+
+namespace claragenomics {
+    std::vector<Overlap> OverlapperTriggered::fuse_overlaps(std::vector<Overlap> unfused_overlaps){
```

Comment:
please add a short comment explaining what the algorithm is doing

Reply:
This is documented in `overlapper_triggered.hpp`. I now added more documentation and an example to the declaration.